### ISSUE TITLE
Configure where to send email notifications

### ISSUE THREAD

**Issue by jmadans** [codeforamerica/cityvoice-survey-builder#88]
Subscribing to email notifications gives users the error page. Posting as a builder issue b/c I'm guessing this is simply a matter of the app not knowing what inbox to hit with the emails. 

So, the app needs to know where to send email addresses.

**Comment by jmadans** [comment]
Another, more expedient fix would be to just remove the email sign up window!

**Comment by daguar** [comment]
Nah, we'd need to also add SendGrid and configure it within the builder. Not a tiny thing. I'd just remove the window for now.
